_posts/en/2020-02-16-what-devops-looks-like.md

```diff
@@ -0,0 +1,122 @@
+
+3. Enabling a continuous improvement environment: As technologies and requirements change rapidly over the course of time the only solution is to be proactive and not reactive. As well as rapid change it is important to accept the complexity of making technology work together. Very rarely will be able to get any service or application architecture exactly right and it is difficult to realize this until an application is actually in production. A significant amount of time should be devoted everyday to see how processes and technology can be improved to alleviate bottlenecks. If you're wondering how this is possible when you are so overburdened with your current work, strategy #2 ( automation ) is supposed to alleviate this so that you can focus on this valuable work.
+
+Let's look at an example of an alternate release process that incorporates these three pillars.
+
+<div style = "display: block; width: inherit; height: 400px; text-align: center;">
+    <a href="/ITStrategy/assets/images/Feature-Intake-to-Release-BPM-Alternate.png" style="display: block;">
+        <img src = "/ITStrategy/assets/images/Feature-Intake-to-Release-BPM-Alternate.png"
+        alt = "Map of alternate release process including which incorporates DevOps methodologies to enable frequent releases. Functions are encapsulated within a single product teams and interactions with central operational teams are through automated checks and tests in incorporated in the build pipeline"
+        style="width: 550px; height: 399px;"
+        />
+    </a>
+    <small> A possible alternate release process ( You can click to expand this image )</small>
+</div>
+<br/>
+
+Already we can see this is subjectively a simpler process to digest and understand. In this process, two weeks of work take a day or less to release. All the functions are encapsulated within one product team. Interaction with functions such as IT Security, Accessibility and Testing Services is through automated checks and processes incorporated into the build. These central operational teams are instead tasked with continually addressing bottlenecks and championing improvements rather than solely a fire fighting role. An incident response role still exists, things will go wrong sometimes regardless of any process or methodology taken. The outcome of these incidents however is not to add an unrealistic burden to operational staff to manually ensure something doesn't happen again, rather fixing the root of the issue by improving the process and incorporating automated checks to ensure that this incident never happens again.
```

Comment:
"Already we can see this is subjectively a simpler process to digest and understand."

I'd remove this sentence. Instead of saying something is subjective, just jump into explaining why this process is more effective (as is done in the following sentences). More succinct without any loss of meaning.